packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -157,32 +164,41 @@ export class HbarLimitService implements IHbarLimitService {
 
   /**
    * Checks if the given eth address or ip address should be limited.
+   *
    * @param {string} mode - The mode of the transaction or request.
    * @param {string} methodName - The name of the method being invoked.
+   * @param {string} txConstructorName - The name of the transaction constructor associated with the transaction.
    * @param {string} ethAddress - The eth address to check.
    * @param {string} [ipAddress] - The ip address to check.
    * @param {string} [requestId] - A prefix to include in log messages (optional).
-   * @param {number} [estimatedTxFee] - The total estimated transaction fee, default to 0.
+   * @param {number} [estimatedTxFee] - The total estimated transaction fee for preemptive HBAR limit logic, default to 0.
    * @returns {Promise<boolean>} - A promise that resolves with a boolean indicating if the address should be limited.
    */
   async shouldLimit(
     mode: string,
     methodName: string,
+    txConstructorName: string,
     ethAddress: string,
     ipAddress?: string,
     requestId?: string,
     estimatedTxFee: number = 0,
   ): Promise<boolean> {
     const requestIdPrefix = formatRequestIdMessage(requestId);
-    if (await this.isDailyBudgetExceeded(mode, methodName, estimatedTxFee, requestIdPrefix)) {
+    this.logger.trace(
+      `${requestIdPrefix} Initiating HBAR rate limit assessment process: txConstructorName=${txConstructorName}, mode=${mode}, methodName=${methodName}`,
+    );
```

Comment:
Still, do we really need this?

Reply:
As mentioned, it might offer some minor benefits for the debugging process. However, considering the already large volume of logs, I believe we can manage without it. Reverted.